src/Raven.Studio/typescript/components/common/shell/setup.ts

```diff
@@ -44,3 +45,38 @@ export function initRedux() {
         topology.nodes.subscribe(onClusterTopologyChanged);
     });
 }
+
+function initYup() {
```

Comment:
please make sure it is executed in storybook and tests, ok ? 

If not - please move to diferent class and init it there.

Reply:
I checked and it works everywhere